### ISSUE TITLE
[generate:profile-stub] Add support for generating profile stubs

### ISSUE THREAD

**Issue by jmolivas** [hechoendrupal/DrupalConsole#295]
Moved from URL at d.o. https://www.drupal.org/node/2392825 created by Sam152 
I think generating stubs for profiles would be great. It would also be really handy to 
have the generated profile be based on one of the core profiles, basically allowing 
you to use standard as a starting point.

```
jmolivas commented 22 days ago:
@Sam152 may you share the documentation link ?, for us to take a look and find out 
is this feature is doable

Sam152 commented 22 days ago:
Here is the profile documentation: https://www.drupal.org/node/2210443

Most profile are forked from "standard" or "minimal" which include quite a bit of 
boilerplate. It might be beyond the scope of this tool to provide a profile based on one 
of these starting points, as there are no special allowances to allow fork-ability, 
but it would be nice.

jmolivas commented 6 days ago:
We achieve an important step in order to be able to work on this issue, to have the 
global installer http://jmolivas.com/install-drupal-console-globally,

We will take a look at the doc link and your blog post 
https://codedrop.com.au/blog/test-driving-drupal-8-writing-install-profile in order to 
work and have a patch/PR for this issue.
```

**Comment by pfrenssen** [comment]
This has been fixed in PR #1406. This can be closed.

**Comment by jmolivas** [comment]
@pfrenssen: Thanks.
